skimage/util/tests/test_apply_parallel.py

```diff
@@ -47,8 +47,7 @@ def test_apply_parallel_lazy():
     # apply the filter on a Dask Array
     result2 = apply_parallel(threshold_local, d, depth=5,
                              extra_arguments=(3,),
-                             extra_keywords={'mode': 'reflect'},
-                             compute=False)
+                             extra_keywords={'mode': 'reflect'})
```

Comment:
I would very much like an additional test where the input is a Dask Array and `compute` is True.

Reply:
Done in `test_apply_parallel`.